src/spandrel/architectures/GFPGAN/arch/gfpgan_bilinear_arch.py

```diff
@@ -268,13 +268,10 @@ def __init__(
             sft_half=sft_half,
         )
 
-        # load pre-trained stylegan2 model if necessary
         if decoder_load_path:
-            self.stylegan_decoder.load_state_dict(
-                torch.load(
-                    decoder_load_path, map_location=lambda storage, loc: storage
-                )["params_ema"]
-            )
+            # Refuse to attempt to load a decoder here
+            raise NotImplementedError(f"Got a non-empty {decoder_load_path=}")
```

Comment:
Why not just remove the arg?

Reply:
Because it'd then be non-obvious why a new model file someone tries is not working as they expect.